src/MagnumPlugins/WavAudioImporter/WavHeader.h

```diff
@@ -31,30 +31,59 @@
 
 #include "Magnum/Types.h"
 
+#define WAVE_FORMAT_PCM 0x0001
+#define WAVE_FORMAT_IEEE_FLOAT 0x0003
+#define WAVE_FORMAT_ALAW 0x0006
+#define WAVE_FORMAT_MULAW 0x0007
+#define WAVE_FORMAT_EXTENSIBLE 0xFFFE
+
 namespace Magnum { namespace Audio {
 
 #pragma pack(1)
+/** @brief RIFF chunk */
+struct RiffChunk {
+    char chunkId[4];            /**< @brief chunk name (4 characters) */
+    UnsignedInt chunkSize;      /**< @brief size of chunk (does not include chunk header) */
+};
+#pragma pack()
+
+#pragma pack(1)
 /** @brief WAV file header */
-struct WavHeader {
-    char chunkId[4];                /**< @brief `RIFF` characters */
-    UnsignedInt chunkSize;          /**< @brief Size of the rest of the file */
+struct WavHeaderChunk {
+    RiffChunk chunk;
     char format[4];                 /**< @brief `WAVE` characters */
+};
+#pragma pack()
 
-    char subChunk1Id[4];            /**< @brief `fmt ` characters */
-    UnsignedInt subChunk1Size;      /**< @brief 16 for PCM */
+#pragma pack(1)
+/** @brief WAV 'fmt' header */
+struct WavFormatChunk {
+    RiffChunk chunk;
     UnsignedShort audioFormat;      /**< @brief 1 = PCM */
     UnsignedShort numChannels;      /**< @brief 1 = Mono, 2 = Stereo */
```

Comment:
I guess the documentation for `audioFormat` and `numChannels` is now outdated?

Also, along with the proposed change of `#define`s to `WavFormat` enum, I'd change the type of `audioFormat` to `WavFormat`, to avoid ugly casts in the code.